Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity.py

```diff
@@ -250,6 +253,78 @@ def purge_ediscovery_data(self, case_id, search_id, purge_type, purge_areas):
         }
         return self.ms_client.http_request(method='POST', url_suffix=url, json_data=body, resp_type='response')
 
+    def create_mail_assessment_request(self, recipient_email, expected_assessment, category, user_id, message_id):
+        url = 'informationProtection/threatAssessmentRequests'
```

Comment:
this variable is showing multiple times in method scope
Please extract it to a global const and change its name to url_suffix